cms/djangoapps/contentstore/helpers.py

```diff
@@ -375,10 +384,30 @@ def _import_xml_node_to_parent(
     if copied_from_block:
         # Store a reference to where this block was copied from, in the 'copied_from_block' field (AuthoringMixin)
         temp_xblock.copied_from_block = copied_from_block
+        # Try to link the pasted block (downstream) to the copied block (upstream).
+        temp_xblock.upstream = copied_from_block
```

Comment:
@bradenmacdonald Currently, this PR sets `.upstream` when applicable _in addition_ to `.copied_from_block`. Should it set `.upstream` _instead of_ `.copied_from_block`? It would make the OLX a little less verbose.

```xml
<html filename="html1" copied_from_block="lb:Kyle:Lib1:html:fbbc03fa-e8b0-4d71-88bf-87fead5f850b" display_name="Text v5" editor="visual" name="null" source_code="null" tags="[]" upstream="lb:Kyle:Lib1:html:fbbc03fa-e8b0-4d71-88bf-87fead5f850b" upstream_display_name="Text v5" upstream_version="5" use_latex_compiler="false"/>
```

Reply:
@kdmccormick I think that makes sense, assuming that if/when we unlink it from the library, we remove `.upstream` and instead set `.copied_from_block` at that time.

Reply:
Great idea. Done, in my latest commit.